weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1293,3 +1329,32 @@ def _combine_conditions(conditions: typing.List[str], operator: str) -> str:
         raise ValueError(f"Invalid operator: {operator}")
     combined = f" {operator} ".join([f"({c})" for c in conditions])
     return f"({combined})"
+
+
+def find_call_descendants_in_memory_dfs(
```

Comment:
nit: prefer to not leak implementation details (dfs) into the function name

Reply:
also, the "in memory" part is presumed since you are passing an in memory list of calls